Splitting on compound names needs to strip leading and trailing spaces

https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/e1bf2644e3e9b77bf40a7af8590495cf97daff2a/DataRepo/utils/accucor_data_loader.py#L473-L475

When loading a new datafile, and pre-loading compounds "2-keto-isocaproate" and "2-keto-methylvalerate" , if a xlsx file (`exp039b3_selected_negative_corrected.xlsx`) contains a complex multi-compound detection joined by slash `'/'` but includes spaces `' / '`, like `2-keto-isocaproate / 2-keto-methylvalerate`, then you get the following error

```
Validating data...
Could not find compound 2-keto-isocaproate 
Could not find compound  2-keto-methylvalerate
Traceback (most recent call last):
  File "/Users/jcmatese/dev/tracebase/./manage.py", line 22, in <module>
    main()
  File "/Users/jcmatese/dev/tracebase/./manage.py", line 18, in main
    execute_from_command_line(sys.argv)
  File "/Users/jcmatese/dev/tracebase/.venv/lib/python3.9/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/Users/jcmatese/dev/tracebase/.venv/lib/python3.9/site-packages/django/core/management/__init__.py", line 413, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/jcmatese/dev/tracebase/.venv/lib/python3.9/site-packages/django/core/management/base.py", line 354, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/Users/jcmatese/dev/tracebase/.venv/lib/python3.9/site-packages/django/core/management/base.py", line 398, in execute
    output = self.handle(*args, **options)
  File "/Users/jcmatese/dev/tracebase/DataRepo/management/commands/load_study.py", line 170, in handle
    call_command(
  File "/Users/jcmatese/dev/tracebase/.venv/lib/python3.9/site-packages/django/core/management/__init__.py", line 181, in call_command
    return command.execute(*args, **defaults)
  File "/Users/jcmatese/dev/tracebase/.venv/lib/python3.9/site-packages/django/core/management/base.py", line 398, in execute
    output = self.handle(*args, **options)
  File "/Users/jcmatese/dev/tracebase/DataRepo/management/commands/load_accucor_msruns.py", line 133, in handle
    loader.load_accucor_data()
  File "/Users/jcmatese/dev/tracebase/DataRepo/utils/accucor_data_loader.py", line 999, in load_accucor_data
    raise e
  File "/Users/jcmatese/dev/tracebase/DataRepo/utils/accucor_data_loader.py", line 987, in load_accucor_data
    self.validate_data()
  File "/Users/jcmatese/dev/tracebase/DataRepo/utils/accucor_data_loader.py", line 165, in validate_data
    self.validate_peak_groups()
  File "/Users/jcmatese/dev/tracebase/DataRepo/utils/accucor_data_loader.py", line 502, in validate_peak_groups
    assert missing_compounds == 0, f"{missing_compounds} compounds are missing."
AssertionError: 2 compounds are missing.
```

Also, we may want to quote-format the validation message so that the spaces are more easily detected, but if they are stripped, perhaps that will no longer be an issue.

--- a/DataRepo/utils/accucor_data_loader.py
+++ b/DataRepo/utils/accucor_data_loader.py
@@ -471,7 +471,10 @@ class AccuCorDataLoader:
                 """
 
                 self.peak_group_dict[peak_group_name]["compounds"] = []
-                compounds_input = [compound_name.strip() for compound_name in peak_group_name.split('/')]
+                compounds_input = [
+                    compound_name.strip()
+                    for compound_name in peak_group_name.split("/")
+                ]
                 for compound_input in compounds_input:
                     try:
                         mapped_compound = Compound.compound_matching_name_or_synonym(

